.github/workflows/review.yml

```diff
@@ -0,0 +1,186 @@
+on:
+  pull_request:
+  workflow_dispatch:
+    inputs:
+      build_required:
+        description: "Build images (true/false)"
+        required: true
+        default: "false"
+      pr_number:
+        description: "Pull request number (if redeploy without build) or own number for environment"
+        required: true
+
+env:
+  CONTAINER_REGISTRY: ghcr.io
+  CONTAINER_REGISTRY_USER: ${{ secrets.GHCR_CONTAINER_REGISTRY_USER }}
+  CONTAINER_REGISTRY_PASSWORD: ${{ secrets.GHCR_TOKEN }}
+  CONTAINER_REGISTRY_REPO: ghcr.io/city-of-helsinki/${{ github.event.repository.name }}
+  REPO_NAME: ${{ github.event.repository.name }}
+  KUBECONFIG_RAW: ${{ secrets.KUBECONFIG_RAW }}
+  BUILD_ARTIFACT_FOLDER: 'build_artifacts'
+  SERVICE_ARTIFACT_FOLDER: 'service_artifacts'
+  BASE_DOMAIN: ${{ secrets.BASE_DOMAIN_STAGING }}
+  DATABASE_USER: user
+  DATABASE_PASSWORD: testing-password
+  K8S_REQUEST_CPU: 50m
+  K8S_REQUEST_RAM: 200Mi
+  K8S_LIMIT_CPU: 500m
+  K8S_LIMIT_RAM: 400Mi
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        service: [ 'backend', 'employer' ]
+        include:
+          - service: backend
+            context: ./backend
+            dockerfile: ./backend/Dockerfile
+            port: 8000
+          - service: employer
+            context: ./frontend
+            dockerfile: ./frontend/employer/Dockerfile
+            port: 3000
+    runs-on: ubuntu-latest
+    name: Build
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Build ${{ matrix.service }}
+        if: github.event_name == 'pull_request' || github.event.inputs.build_required == 'true'
+        uses: andersinno/kolga-build-action@v2
+        env:
+          DOCKER_BUILD_ARG_NEXT_PUBLIC_BACKEND_URL: https://helsinkibenefit-backend-5.test.kuva.hel.ninja
+          DOCKER_BUILD_SOURCE: ${{ matrix.dockerfile }}
+          DOCKER_BUILD_CONTEXT: ${{ matrix.context }}
+          DOCKER_IMAGE_NAME: ${{ matrix.service }}
+          SERVICE_PORT: ${{ matrix.port }}
+  review:
+    strategy:
+      fail-fast: false
+      matrix:
+        service: [ 'backend', 'employer' ]
+        include:
+          - service: backend
+            context: backend
+            dockerfile: ./backend/Dockerfile
+            database: true
+            port: 8000
+          - service: employer
+            context: frontend
+            dockerfile: ./frontend/employer/Dockerfile
+            database: false
+            port: 3000
+    runs-on: ubuntu-latest
+    needs: build
+    name: Review
+    steps:
+      - uses: actions/checkout@v2
+      - uses: andersinno/kolga-setup-action@v2
+        with:
+          pr_number: ${{ github.event.inputs.pr_number }}
+      - name: Review-Services
+        if: matrix.database
+        uses: andersinno/kolga-deploy-service-action@v2
+        with:
+          projects: ${{ github.event.repository.name }}-${{ matrix.service }}
+        env:
+          POSTGRES_IMAGE: "docker.io/andersinnovations/postgis:11-bitnami"
```

Comment:
Maybe switching to Postgres 12 here as we use version 12 for development

Reply:
The aiven postgress service is configured to use version 11.11 in staging + production. Won't that cause problems?

Reply:
Hmn, if so would it make more sense for us to use 11.11 in development too?

Reply:
Maybe, at least then there won't be any surprises between local, staging / production. I'm not that familiar with the databases and how well they are backwards compatible.